### ISSUE TITLE
RandAugment usage

### ISSUE THREAD

**Issue by rccchoudhury** [zhaoyue-zephyrus/AVION#8]
I noticed that RandAugment is only used in the Kinetics dataloader if fast_rrc is off. Does this mean that RandAugment was not used for pre-training or finetuning? I also noticed even if RandAugment is moved to the GPU along with other transforms, data loading speed is quite a bit slower. Have you seen this issue before?

**Comment by zhaoyue-zephyrus** [comment]
Hi @rccchoudhury ,

 Great point. Yes, `RandAugment` is not supported in the fast-loading mode. And yes It is not used for pre-training. For more detail, please refer to Table 9 of the supplementary material of [VideoMAE](https://proceedings.neurips.cc/paper_files/paper/2022/file/416f9cb3276121c42eebb86352a4354a-Supplemental-Conference.pdf). My hunch is that 80-90% masking also serves as sufficiently strong augmentation and you don't need this for pre-training phase. 

For fine-tuning, it is used instead. RandAugment indeed takes more time than regular augmentations whether it is done on the CPU or GPU end. However, fine-tuning usually takes much fewer epochs (50~100 epochs) than pre-training (800~1600 epochs). Therefore, keeping `RandAugment` in the fine-tuning stage won't affect the overall speedup.

Hope this clarification helps.

Best,
Yue
